krausening-python/src/krausening/logging/log_manager.py

```diff
@@ -26,9 +36,14 @@ def __init__(self):
         else:
             LogManager.__instance = self
 
-    def get_logger(self, name: str) -> logging.Logger:
+    def _get_log_level(self, log_level: str) -> int:
```

Comment:
This set of code allows users to specify the logging level rather than only being able to use the `logging.INFO` level.

Reply:
C: Addresses issue #30